pesacheck_meedan_bridge/py/main.py

```diff
@@ -1,16 +1,22 @@
 import json
 import sys
 
+import lxml.html
 import requests
 import sentry_sdk
 import settings
 from check_api import post_to_check
 from database import PesacheckDatabase, PesacheckFeed
-from trafilatura import extract
 
 
-def html_to_text(content):
-    return extract(content, include_links=True, include_images=True) or content
+def extract_summary(content):
+    tree = lxml.html.fromstring(content)
+    figures = tree.xpath("//figure")
+    if len(figures) == 0:
+        return None
+    summary_el = figures[0].getprevious()
+    summary_text = lxml.html.tostring(summary_el, method="text", encoding="unicode")
```

Comment:
Is this different from [`text_content()`](https://lxml.de/lxmlhtml.html#html-element-methods), i.e.
```python
    summary_text = summary_el.text_content()
```